src/SASViyaApiClient.ts

```diff
@@ -380,6 +396,9 @@ export class SASViyaApiClient {
    * @param accessToken - an access token for an authorized user.
    * @param sessionId - optional session ID to reuse.
    * @param silent - optional flag to turn of logging.
+   * @param data - execution data.
+   * @param debug - flag taht indicates debug mode.
+   * @param expectWebout - flag that indicates that web output is expected
```

Comment:
When set to `true`, the automatic `_webout` fileref will be checked for content, and that content returned.  This fileref is used when the Job contains a SASjs web request (as opposed to executing arbitrary SAS code).